app/lib/methods/getRooms.ts

```diff
@@ -0,0 +1,12 @@
+import { IRocketChat } from '../../definitions/IRocketChat';
+
+export default function (this: IRocketChat, updatedSince: Date) {
+	let updatedDate = '';
```

Comment:
I did this because TypeScript complains about it being two different types, we are receiving a `Date` (and we know that because we're using `toISOString()`, but at the same time we want to assign that value to our variable, which is already `Date`. On cases like this nor `Date | string` or `Date & string` work.

Reply:
Yep.
Javascript lets you things like that 🙈